org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/serializer/SyntacticSequencerTestLanguage.ecore

```diff
@@ -30,17 +30,19 @@
         containment="true"/>
     <eStructuralFeatures xsi:type="ecore:EReference" name="x15" eType="#//FragmentCallerType"
         containment="true"/>
+    <eStructuralFeatures xsi:type="ecore:EReference" name="x16" eType="#//Bug398890"
+        containment="true"/>
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="MandatoryKeywords">
-    <eStructuralFeatures xsi:type="ecore:EAttribute" name="val1" eType="ecore:EDataType platform:/resource/org.eclipse.emf.ecore/model/Ecore.ecore#//EString"/>
-    <eStructuralFeatures xsi:type="ecore:EAttribute" name="val2" eType="ecore:EDataType platform:/resource/org.eclipse.emf.ecore/model/Ecore.ecore#//EString"/>
-    <eStructuralFeatures xsi:type="ecore:EAttribute" name="val3" eType="ecore:EDataType platform:/resource/org.eclipse.emf.ecore/model/Ecore.ecore#//EString"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="val1" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
```

Comment:
this change from platform:/resource/ to http://www is suspicious and should not happen. Did you generate via 

    /org.eclipse.xtext.tests/src/org/eclipse/xtext/GenerateAllTestLanguages.mwe2

or via 

    /org.eclipse.xtext.tests/src/org/eclipse/xtext/serializer/GenerateSerializerLanguages.mwe2

?